analyzer-comments/python/general/no_return.md

```diff
@@ -1,8 +1,9 @@
 # general missing return statement
 
-A `return` statment is not being used in this function to return the result string, and this code should have failed one or more test cases during testing.
+A `return` statement is not being used in this function to return results.
+ This code should have failed one or more test cases during testing.
 
-Please review your test results.
+Please review the test results, and edit the code accordingly.
 
-If you have not yet run the tests yet for this exercise, please do so now by issuing the command ```run pytest``` inside the `two-fer` exercise directory.
+If you have not yet run the tests for this exercise, please do so now by issuing the command ```run pytest``` inside the problem exercise directory (e.g. `exercism/python/<exercise-name>/`).
```

Comment:
I'm not sure if this comment makes sense anymore in v3. AFAIK the analyzer doesn't even run if the test runner reported failure.

Reply:
Very good point.  I will delete this.